### PR TITLE
Fix some grammatical errors

### PR DESCRIPTION
**Pull request by PVasileff**

### CODE REVIEW

File: localization/en_US.inc [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <?php
 $labels = array();
-$labels['automaticallycollected']  = 'Automatically Collected';
-$labels['useautoabook']  = 'Use automatic address book';
-$labels['useforcompletion'] = 'Use automatically collected addresses for autocompletion';
-$labels['contactremoved'] = 'Contact(s) deleted from automatic address book';
-$labels['contactmoved'] = 'Contact(s) moved from automatic address book';
+$labels['automaticallycollected']  = 'Автоматично събрани адреси';
```
Comment (sblaisot) [inline localization/en_US.inc]: please don't include bulgarian translations in en_US file. move them in bg_BG instead.

**Comment by sblaisot** [comment]
Thanks a lot for your PR. Unfortunately, some bulgarian translations are included in the en_US file. Can you update that to have all translations un bg_BG file ?

**Comment by PVasileff** [comment]
Problem with en_US was fixed. Sorry about that.
Bulgarian Updated.

**Comment by sblaisot** [comment]
Thanks. LGTM.

**Pull request merged**